### DISCUSSION TITLE
YouTube + ByeDPI в AppImage

### DISCUSSION THREAD

**Discussion by prontok** [hufrea/byedpi#173] [Show and tell]
Завернул сайт https://youtube.com/ в chrome-sandbox и совместил со ByeDPI, и все это упаковал в AppImage.
Теперь можно просто запустить YouTube в виде готового приложения на рабочем столе, которое сразу будет поддерживать ByeDPI из коробки.

Собранный AppImage поддерживает:
1. ByeDPI + YouTube в виде отдельного приложения. После завершения работы приложения ByeDPI автоматически останавливается и не держит порт.
2. Содержит metainfo, тем самым AppImage поддерживает обновления *.zsync, которые будут зависеть от выпусков [ByeDPI](https://github.com/hufrea/byedpi)

Загрузить можно отсюда: https://github.com/prontok/YouTubeDPI

**Comment by djboow** [comment]
> Загрузить можно отсюда: https://github.com/prontok/YouTubeDPI

Отличная сборка. Москва - т2(ростелеком) показывает без проблем.

Напишите, пожалуйста, какие атрибуты вы прописали у ByeDPI в этой сборке.

**Reply by prontok** [reply]
Внутри AppImage для ByeDPI я использую такие атрибуты:
./ciadpi -i 127.0.0.1 -p 8080 --oob 1 -A t

**Comment by Khlestkin** [comment]
С провайдером МТС работает, спасибо.
